api/rest/stat_series.md

```diff
@@ -34,69 +35,200 @@ You can find a list of StatisticalVariables with human-readable names [here](/st
 - `unit`: The dcid of the preferred `unit` value.
 - `scaling_factor`: The preferred `scalingFactor` value.
 
-## GET Request
+## How to construct a request to the place statistics time series endpoint
 
-**Examples**
+### Step 1: Assembling the information you will need
 
-```bash
-curl 'https://api.datacommons.org/stat/series?place=geoId/06&stat_var=Count_Person_Male'
-```
+This endpoint requires two arguments and offers four additional optional arguments, as listed:
 
-<iframe width="100%" height="300" src="//jsfiddle.net/datacommonsorg/w32gmo68/8/embedded/" allowfullscreen="allowfullscreen" allowpaymentrequest frameborder="0"></iframe>
+ - `place`: For this parameter, you will need to specify the DCID (the unique ID assigned by Data Commons to each node in the graph) of the place you are interested in.
+ - `stat_var`: The statistical variable whose value you are interested in.
+
+In addition to these required properties, this endpoint also allows for other, optional arguments.
+
+  - `date`: If the property queried only takes on node values, you can use this argument to filter nodes in the response, ensuring the response only contains nodes with the specified type.
+
+  - `measurement_method`: You can specify this argument as `out` to indicate that you desire the response to only include nodes which are supercategories of the specified `DCIDs`, or `in` to only return nodes that are subcategories of the specified `DCIDs`. (For example, South America is a supercategory of Argentina, which in turn is a supercategory of Buenos Aires, as illustrated in Figure 1.)
+  
+  - `observation_period`: (≤ 500) Maximum number of values returned per node.
```

Comment:
@KilimAnnejaro  I think these descriptions are leftover from prop-vals?

remove date and add scalingFactor?

Reply:
sorry about that--fixed!